CONTRIBUTING.md

```diff
@@ -79,19 +79,21 @@ conda install sphinx
 Or, if using ``pip``, you can grab it with
 
 ```bash
+python -m pip install git+https://github.com/spyder-ide/spyder-docs-sphinx-theme.git
```

Comment:
This should be repeated under the `conda` section.

Furthermore, it should presumably follow `pip install sphinx` given Sphinx is an implicit or explicit dependency of the theme.

Is there a reason `python -m pip` is used for this step, yet `pip` is used for the next? We should be consistent and use one or the other.

Reply:
All looks good, thanks @dalthviz !